### PR TITLE
DOCSP-38184 Indicate that any reshardCollection operations are disallowed

### PR DESCRIPTION
**Pull request by kanchana-mongodb**
- [JIRA](https://jira.mongodb.org/browse/DOCSP-38184)
- [Stage](https://preview-mongodbkanchanamongodb.gatsbyjs.io/cluster-sync/DOCSP-38184/reference/limitations/#sharded-clusters)
- [Build](https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=660d60663a5920a29196b65e) (contains errors and warnings nor introduced by changes in this PR)

### CODE REVIEW

File: source/reference/limitations.txt [hunk 1]
```diff
@@ -112,8 +112,8 @@ Sharded Clusters
 
 - The shard key cannot be :ref:`refined <shard-key-refine>` while
   syncing.
-- The shard key cannot be modified using :dbcommand:`reshardCollection`
-  during syncing.
+- The :dbcommand:`reshardCollection` operations from the source cluster
```
Comment (sarah-olson-mongodb) [inline source/reference/limitations.txt:115]: (nit -- minimalism)
Super non blocking. You can omit "The" if you like.

**Review by sarah-olson-mongodb — APPROVED** [review]
Thanks so much @kanchana-mongodb! LGTM with one minor suggestion for consideration.

File: source/reference/limitations.txt [hunk 2]
```diff
@@ -112,8 +112,8 @@ Sharded Clusters
...
+  are not supported.
```
Comment (ajayvijayakumar123) [inline source/reference/limitations.txt:116]:
```suggestion
  are not supported during sync. 
```

**Review by ajayvijayakumar123 — CHANGES_REQUESTED** [review]
One suggestion!

**Review by ajayvijayakumar123 — APPROVED** [review]
LGTM!

File: source/reference/limitations.txt (hunk 1 above)
Comment (kanchana-mongodb) [reply source/reference/limitations.txt:115]: Keeping "The" to be consistent with the other bullet items. Thanks!

**Pull request merged**